scene/resources/style_box_flat.cpp

```diff
@@ -234,6 +234,79 @@ inline void set_inner_corner_radius(const Rect2 style_rect, const Rect2 inner_re
 	inner_corner_radius[3] = MAX(corner_radius[3] - MIN(border_bottom, border_left), 0); // Bottom left.
 }
 
+inline void set_corner_scale(const Rect2 style_rect, const Rect2 inner_rect, const real_t corner_radius[4], Point2 *inner_scale) {
```

Comment:
```suggestion
inline void set_corner_scale(const Rect2 &style_rect, const Rect2 &inner_rect, const real_t corner_radius[4], Point2 *inner_scale) {
```
(I know other methods don't do that, but they should)

Reply:
Ok I pushed the change